.circleci/config.yml

```diff
@@ -0,0 +1,75 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+
+common_steps: &common_steps
+  - checkout
+  - run: gem update --system
+  - restore_cache:
+      key: gem-cache-{{ .Branch }}-{{ checksum "Gemfile" }}
```

Comment:
I guess this should include the gemspec checksum as well, if possible?

Reply:
Ah true, using the gemspec makes more sense here. The original snippet used the gemfile.lock which doesn't work for gems. Will change that.

Reply:
I guess using both would be best? If that's possible. Not sure how this caching works exactly on circleci, but modifying Gemfile without cache invalidation sounds prone to issues.

Reply:
it's just the gem cache so IMHO the gemspec alone is enough